pkg/pbq/store/wal/segment.go

```diff
@@ -124,13 +127,14 @@ func openOrCreateWAL(id *partition.ID, opts *store.StoreOptions) (*WAL, error) {
 			return nil, err
 		}
 		wal = &WAL{
-			fp:          fp,
-			openMode:    os.O_RDWR,
-			wOffset:     0,
-			rOffset:     0,
-			readUpTo:    stat.Size(),
-			partitionID: id,
-			opts:        opts,
+			fp:                fp,
+			openMode:          os.O_RDWR,
+			wOffset:           0,
+			prevSyncedWOffset: 0,
```

Comment:
You may need to set `prevSyncedWOffset` equal to `wOffset` when we finish bootstrap.

Reply:
Fixed this. Thanks!